modules/database/database.bal

```diff
@@ -155,6 +155,13 @@ public type ExperimentFull record {|
     *Experiment;
 |};
 
+# Record containing the template id of an experiment
+#
+# + templateId - template id
+public type Template record {|
+    string? templateId = ();
```

Comment:
Also add the template id to the standard experiment record (to avoid extra requests to get the template id)